server/Arcadia.Assistant.Server.Console/appsettings.json

```diff
@@ -28,6 +28,12 @@
       "Body": "Dear Sir/Madam,\nNotify you, that employee {0} went on sick leave for {1}\nYours faithfully,\nTesting team"
     }
   },
+  
+  "Organization": {
+    "RefreshInformation": {
+      "interval": 10
```

Comment:
how does user know if its minutes, hours, seconds?

Reply:
lets rename to intervalMinutes